tests/test_queries.py

```diff
@@ -310,9 +310,46 @@ def test_currency(client: FlaskClient) -> None:
     json = qmcall(client, {"q": "hvað eru tíu þúsund krónur margir dalir"}, "Currency")
     assert re.search(r"^\d+(,\d+)?$", json["answer"]) is not None
 
+    json = qmcall(
+        client,
+        {"q": "hvað eru þrettán hundruð fjörutíu og sex krónur margir dalir"},
+        "Currency",
+    )
+    assert re.search(r"^\d+(,\d+)?$", json["answer"]) is not None
```

Comment:
Consider adding a check that the number is not only present, but also correct?

Reply:
currency.py currently doesn't return the amount asked for in its answer, so I added tests that compared the answer to the same question where the number wasn't written in natural language :+1:.